packages/react-ui/src/widgets/WrapperWidgetUI.d.ts

```diff
@@ -34,6 +34,7 @@ export type WrapperWidgetUIProps = {
   isLoading?: boolean;
   disabled?: boolean;
 
+  headerItems?: ReactNode;
   actions?: WrapperWidgetAction[];
```

Comment:
@aaranadev @menusal We think actions / options props are too prescriptive. We would like to consider deprecation of those in the future, replaced by the use of this new 'headerItems' prop